zng/docs/spec.md

```diff
@@ -43,17 +46,15 @@ of heterogeneously typed records, e.g., structured logs, where filtering and
 analytics may be applied to a stream in parts without having to fully deserialize
 every value.
 
-ZNG has both a text form called "TZNG",
-comprised of a sequence of newline-delimited UTF-8 strings,
-as well as a binary form called "ZNG".
+ZNG has a binary form called "ZNG" as well as text form called "TZNG" that's
+comprised of a sequence of newline-delimited UTF-8 strings.
 
 ZNG is richly typed and thinner on the wire than JSON.
-Like [newline-delimited JSON (NDJSON)](http://ndjson.org/),
-the ZNG text format represents a sequence of data objects
+ZNG strikes a balance between the narrowly typed but flexible
+[newline-delimited JSON (NDJSON)](http://ndjson.org/) format and
+a more structured approach like [Apache Avro](https://avro.apache.org).
+Like NDJSON, the TZNG text format represents a sequence of data objects
```

Comment:
I've re-ordered the text here to match the flow of the overall doc: We lead with binary ZNG because it's the more compact/efficient representation & the default output of `zq`.